### ISSUE TITLE
Wrong twig filter for rest filter description

### ISSUE THREAD

**Issue by AlexeyKupershtokh** [nelmio/NelmioApiDocBundle#346]
I noticed that the description field for filter params breaks Russian language.
E.g. the string "Идентификатор" is transformed into "\u0418\u0434\u0435\u043d\u0442\u0438\u0444\u0438\u043a\u0430\u0442\u043e\u0440".
I believe that this is because the following twig filter used for filter fields output:

``` html
<td>{{ value|json_encode|replace({'\\\\': '\\'})|trim('"') }}</td>
```

Any idea why is json_encode used there? 

There are no problems with Requirements or Parameters because both of them are using this following template snippet:

``` html
<td>{{ infos.description is defined ? infos.description : ''}}</td>
```

**Comment by DonPrus** [comment]
@willdurand https://github.com/nelmio/NelmioApiDocBundle/pull/775

**Comment by dbu** [comment]
fixed in #775
